packages/calling/src/api.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable import/no-unresolved */
+import {CallHistory} from 'CallHistory/CallHistory';
```

Comment:
Please fix the paths. VSCode makes this mistake of putting in the wrong path which will fail during usage somewhere else.

Reply:
Corrected it